tests/starlette/hello_activity_test.py

```diff
@@ -0,0 +1,45 @@
+import uuid
```

Comment:
By the way I don't mind, since this is just a hello workflow from somewhere else, that you just get rid of these copy/pasted tests. Having said that, showing how to use Starlette-based tests would be nice.